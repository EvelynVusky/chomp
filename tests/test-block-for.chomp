int main() {
    int b = 5;
    {
        int i = 0;
        for (i = 0; i < 10; i = i + 1) {
            b = b + 2;
        }

        string hi = "hello";
        for (i = 5; i < 10; i = i + 1) {
            println(hi);
        }
    }
    println(b);

    return 0;
}