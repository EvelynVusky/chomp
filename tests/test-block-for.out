hello                           
hello
hello
hello
hello
25